helm_deploy/values-train.yaml

```diff
@@ -1,12 +1,21 @@
 ---
 # Per environment values which override defaults in hmpps-locations-inside-prison-api/values.yaml
+environment:
```

Comment:
Didn't know if this was necessary or helm already new the environment ? @mattops

Reply:
You'll need it, helm doesn't know.